### PR TITLE
add option geld-verdienen

### PR DESCRIPTION
**Pull request by HenriettePlane**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2623)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: dutch/bos/ontvoering/meehelpen/klus/geld-verdienen.md [hunk 1]
```diff
@@ -2,4 +2,8 @@ In eerste instantie geloven de mannen je niet maar als ze je Rolex om je arm
 zien zijn ze snel geïnteresseerd. Wie is deze man?
 
 De mannen kijken elkaar aan en na een korte blik vraagt de langste man:
-Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
\ No newline at end of file
+Voor wie werk jij dan eigenlijk en heb je nog meer klussen?
+
+ja, zeg je, en glimlacht duister. Ik bel mijn baas wel even of hij nog wat voor 
+jullie in petto heeft. Je pakt je telefoon, maar je belt helemaal niet je baas,
+je bent bezig de [politie te bellen](politie/politie.md)
```
Comment (casey-collab) [inline dutch/bos/ontvoering/meehelpen/klus/geld-verdienen.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
It looks like the files `dutch/bos/.DS_Store`, `dutch/bos/ontvoering/meehelpen/plassen/.DS_Store`, `dutch/bos/ontvoering/meehelpen/klus/.DS_Store`, `dutch/.DS_Store`, `dutch/bos/ontvoering/meehelpen/.DS_Store`, `dutch/bos/ontvoering/AllesKomtGoed/.DS_Store`, and`dutch/bos/ontvoering/.DS_Store` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@HenriettePlane, hopefully Casey's comment wasn't too confusing. I am guessing the `.DS_Store` files were added by accident, as they are automatically generated by Macs. You can remove them using the command `git rm`.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**